linera-chain/src/unit_tests/chain_tests.rs

```diff
@@ -112,7 +115,7 @@ async fn test_block_size_limit() {
     let mut chain = ChainStateView::new(chain_id).await;
 
     // The size of the executed valid block below.
-    let maximum_executed_block_size = 685;
+    let maximum_executed_block_size = 720;
```

Comment:
TODO: need to investigate this.

Reply:
That's exactly as much as  the `33 bytes`  for `Secp256k1PublicKey` + `2 bytes` for length prefix but I don't know where that'd come from.

Reply:
I'll do that if I don't figure this out before merging.

Reply:
OK, I think where this might come from - there's a `ExecutionOutcome::System(SystemMessage::CreateCommittee(Committee)))` and that innermost struct contains `ValidatorState` which now also gets `AccountPublicKey`.